src/hashtoolkit.py

```diff
@@ -0,0 +1,22 @@
+import cloudscraper, requests

```

Comment:
Do we have to bypass cloudflare? Does it break hashtoolkit's ToS to do this? This software will be used in production so companies may not like this :(

Reply:
Unfortunately, that's the only way I could find of doing this, They dont have any API and I couldn't find a way to access it with requests without bypassing cloudfare :(